3p/ampcontext.js

```diff
@@ -43,6 +43,13 @@ export class AbstractAmpContext {
     /** @private {?string} */
     this.cachedFrameName_ = this.win_.name || null;
 
+    // This variable keeps keeps track when an invalid resize request is made, and
+    // is associated with each iframe. If the request is invalid, then a new request
+    // cannot be made until a certain amount of time has passed, 500 ms by default
+    //(see msecRepeatedRequestDelay). Once the timer has cooled down, a new request can be made.
+    /** @private {?boolean} */
+    this.blockRepeatedInvalidRequests_ = false;
```

Comment:
I feel the better option would be to retain a property called `this.lastRejectedResizeTime_`, so we do not have to do `setTimeout` proactively. Every time we get a new request, we do a timestamp comparison to see whether we should reject.

Reply:
Sounds good, I'll go ahead and make that property.